src/modules/Tasks/components/TasksForm/TasksForm.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { useForm } from 'react-hook-form';
+import { toast } from 'react-toastify';
+import axios from 'axios';
+import './TaskForm.css';
+import '../TasksList/TasksList.css';
+import { FloatingLabel } from 'react-bootstrap';
+import TasksList from './../TasksList/TasksList';
+
+interface TaskFormData {
+    title: string;
+    description: string;
+    employeeId: number;
+    projectId: number;
+}
+
+export default function TaskForm() {
+    const {
+        register,
+        handleSubmit,
+        formState: { isSubmitting, errors },
+        setValue,
+    } = useForm<TaskFormData>({ mode: "onChange" });
+
+    const [employees, setEmployees] = useState<any[]>([]); // Adjust type as necessary
```

Comment:
We need to add the type based on the backend response. Using any is not recommended.